kvm.rb

```diff
@@ -2,8 +2,10 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
-  url "https://github.com/aspnet/Home.git", :branch => 'master'
+  version "1.0.0-beta3"
+  url "https://github.com/aspnet/Home.git", :tag => 'v1.0.0-beta3'
+
+  head "https://github.com/aspnet/Home.git", :branch => 'dev'
```

Comment:
This more closely follows the pattern of other Formulae. A couple notes:

1. The tag 'v1.0.0-beta3' does not currently exist. We should not merge this change until it is created, which will happen at beta3 final release time
2. When you install via "brew install kvm --HEAD" you get the `dev` branch, and it has a special version number "HEAD" which is treated as the absolute latest version, so the version is irrelevant in that case.